src/layout_components.py

```diff
@@ -78,6 +71,11 @@ def get_artist_section():
             ),
             html.Br(),
             html.Br(),
+            html.Br(),
+            html.Br(),
+            html.Br(),
```

Comment:
Jenn I feel the multiple break lines makes the app look off . I feel its better to let go off the minute alignment issue which was there previously than giving a lot of white space .

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/37771404/156851399-0bc363e2-cb59-480d-9968-726c2d9d18ee.png">

Reply:
To add on, We can maybe change the layout to make a row for the widget section and then 2 rows for the plots ( with 1st row for Top artists and 2nd row having 2 columns ( 1 for Trend plot and 1 for the Histogram ) 

I can maybe create an Issue for this and tag it as enhancements for now.

Reply:
Not sure if this happening for this PR or #36 . Tagging @xiangwxt for the same. 

Did the deployment app for PR36 look fine before merging @jennifer-hoang ?

Reply:
Thanks for the feedback. I've removed the additional break lines.

The app looks okay in full size for me, but I did see the change with resizing the window after #36 
![Screen Shot 2022-03-04 at 2 48 44 PM](https://user-images.githubusercontent.com/94317883/156852448-7acba8fb-2248-444e-b58f-bb90c468d87a.png)

Reply:
Yes. Now it's our decision to make , since not everyone will view it in full screen right ? I have fix either way . Let me merge this in main and make another PR.